apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -134,6 +135,13 @@ render self@{ state, setState } =
                    Nothing   -> loginView self
                ]
          }
+   noMatchRoute =
+      element
+        Router.route
+          { exact: false
+          , path: toNullable Nothing
```

Comment:
Just asking for lack of knowledge: why here we pass `Nothing`? What happens if we pass `Just "/"`

Reply:
Then it would match the route `"/"` (which we already handle), but we want it to match everything if all other routes fail.

The router will choose the first one to match from this array of routes:
```
Router.switch { children: [ paymentListRoute, mittKontoRoute, noMatchRoute ]
```
By passing `path: null` to the `Route` component, it matches everything as it says in the documentation: https://reactrouter.com/web/api/Route/path-string-string

Reply:
We should probably use `Redirect` here (to root), although it's not yet implemented in our router wrapper 😄